### PR TITLE
Spelling: rework

### PR DESCRIPTION
**Pull request by comradekingu**
"at work" consistent with welcome_feature_1_text in Android

### CODE REVIEW

**Comment by marinofaggiana** [comment]
@rakekniven commit ?

File: iOSClient/Supporting Files/en.lproj/Intro.strings [hunk 1]
```diff
@@ -9,13 +9,13 @@
 // Intro
 
 "_intro_1_title_"   = "A safe home for all your data";
-"_intro_1_text_"    = "Access, share and protect your files at home and in your enterprise";
+"_intro_1_text_"    = "Access, share and protect your files at home and at work";
 
 "_intro_11_title_"  = "Use gestures in files view";
-"_intro_11_text_"   = "Swipe on right or on left for options or a long touch for Copy Paste Open in… or press for 3D Touch, when available";
+"_intro_11_text_"   = "Swipe → or ←: Options \n\nLong touch: Copy/Paste or Open in… \n\nPress: 3D Touch (when available)";
```
Comment (rakekniven) [inline iOSClient/Supporting Files/en.lproj/Intro.strings]: @marinofaggiana Will "\n" work inside string on iOS?

Text changes are ok for me.

Comment (enoch85) [inline iOSClient/Supporting Files/en.lproj/Intro.strings]: Ping @marinofaggiana

**Comment by marinofaggiana** [comment]
@rakekniven not use master, only develop

**Comment by rakekniven** [comment]
@marinofaggiana Sorry! Could you merge this changes to "develop"? Thank you.

**Pull request merged**